src/Commands/BuildToolsBase.php

```diff
@@ -778,7 +783,7 @@ public function pushCodeToPantheon(
         $this->passthru("git -C $repositoryDir add --force -A .");
 
         // Now that everything is ready, commit the build artifacts.
-        $this->passthru("git -C $repositoryDir commit -q -m 'Build assets for $env_label.'");
+        $this->passthru("git -C $repositoryDir commit -q -m '$message'");
```

Comment:
Need to appropriately escape $message in case it contains `'`

Reply:
Good call.